src/precice/Tooling.cpp

```diff
@@ -34,6 +36,7 @@ void checkConfiguration(const std::string &filename, const std::string &particip
       0,
       size};
   xml::configure(config.getXMLTag(), context, filename);
+  fmt::print("{}: {}\n", filename, fmt::styled("correct", fmt::emphasis::bold | fg(fmt::color::green)));
```

Comment:
That's nice! We could, however, give some hint about the scope of the checking. If people get a "correct" here and an error during their actual simulation, this may be confusing.

Reply:
Could you propose something? I am not sure how to phrase this well.

Reply:
Maybe just add some very short description in the beginning on what this tool does. For example:

```
$ precice-tools check valid-config.xml
Checking valid-config.xml for syntax and basic setup issues... (see https://precice.org/tooling-builtin.html)
valid-config.xml: correct
```

Reply:
Good idea. I'll leave the link out though.

Reply:
See OP for new message.